jobmaster/system/impl.go

```diff
@@ -310,39 +308,33 @@ func (m *Master) addScheduleTasks(group scheduleGroup) {
 }
 
 func (m *Master) monitorRunningTasks() {
+	timer := time.NewTicker(time.Second * 1)
+	defer timer.Stop()
 	for {
 		select {
 		case <-m.ctx.Done():
 			return
-		default:
+		case <-timer.C:
+			m.checkRunningTasks()
 		}
-		delay := m.monitorTasksLimit.Reserve().Delay()
-		if delay != 0 {
-			log.L().Logger.Warn("monitor task rate limit", zap.Duration("delay", delay))
```

Comment:
I copied this log from reschedule logic. How about just removing it

Reply:
Seems a ticker or rate limiter has the same effect.

Reply:
Yeah, that's true.